internal/archive/archive_test.go

```diff
@@ -302,6 +302,38 @@ func testArchive(t *testing.T, when spec.G, it spec.S) {
 					verify.nextSymLink("/nested/dir/dir-in-archive/sub-dir/link-file", "../some-file.txt")
 				}
 			})
+
+			when.Focus("files are compressed in fat (MSDOS) format", func() {
```

Comment:
Need to remove this focus :)

Reply:
Good point! We should probably add automation to ensure there isn't a Focus...